packages/node-runtime-worker-thread/package.json

```diff
@@ -29,9 +29,11 @@
     "webpack-build": "npm run compile && webpack --mode production",
     "webpack-build-dev": "npm run compile && webpack --mode development",
     "compile": "tsc -p tsconfig.json",
-    "prepublish": "npm run webpack-build",
+    "prepublish": "npm run webpack-build && npm run create-purls-file",
```

Comment:
@addaleax I simplified the whole setup to minimize changes and just going to append the purls file into `dist/purls.txt` of the package since that seems to be all we need. I assume we don't need to append `.mongosh_crypt_...` as I don't see the files for it there.

Reply:
> I assume we don't need to append `.mongosh_crypt_...` as I don't see the files for it there.

Right, we only include the crypt_shared library in the distributed Compass and mongosh packages but not the individual npm packages in the monorepo 👍